docs/source/ecf/guidance.html.md

```diff
@@ -74,11 +74,11 @@ For more detailed information see the specifications for this [view school detai
       "attributes": {
         "name": "School Example",
         "urn": "123456",
-        "cohort": 2024,
+        "cohort": 2025,
```

Comment:
not sure if we should be amending this yet or when the fixes are in place, I'd suggest we wait till the spec is updated, let me know your thoughts